shido/chain.json

```diff
@@ -57,18 +57,18 @@
     },
     "versions": [
       {
-        "name": "v1.0.0",
-        "tag": "v1.0.0",
-        "recommended_version": "v1.0.0",
+        "name": "v2.0.0",
```

Comment:
Please retain v1, and add v2 as a new version object later in the `versions` array

Reply:
Done